docs/Field.md

```diff
@@ -331,100 +331,55 @@ demo.
 							<label class="cui__field__label">
 								Name
 							</label>
-							<input
-										class="cui__field__input"
-										oninput="floatingLabel(this)"
-										onfocus="dofocus(this)"
-										onblur="doblur(this)"
-								/>
+							<input class="cui__input__input" />
```

Comment:
But this is the showcase for the `.cui__field`, so I guess we should be using `.cui__field__input`.